include/mbedtls/x509_crt.h

```diff
@@ -175,19 +175,34 @@ int mbedtls_x509_crt_parse_der( mbedtls_x509_crt *chain, const unsigned char *bu
                         size_t buflen );
 
 /**
- * \brief          Parse one or more certificates and add them
- *                 to the chained list. Parses permissively. If some
- *                 certificates can be parsed, the result is the number
- *                 of failed certificates it encountered. If none complete
- *                 correctly, the first error is returned.
+ * \brief          Parse one DER-encoded or one or more concatenated PEM-encoded
+ *                 certificates and add them to the chained list.
+ *
+ *                 For PEM-encoded CRTs, the function parses permissively:
```

Comment:
In english, you can use a colon in this way, but the second clause can't begin with a capital letter. It should be:

> For PEM-encoded CRTs, the function parses permissively: if at least one certificate can be parsed, the function returns the number of certificates for which parsing failed (hence `0` if all certificates were parsed successfully).

It's a bit unusual though, and you rarely see colons used this way in the written language (because people don't know how to use them), so the sentence may be clearer if it's rephrased to remove it.

Reply:
@sbutcher-arm Will fix it.

Reply:
@sbutcher-arm Changed in e8658e2.